### ISSUE TITLE
✨ Feature parity: Customfeed or Multireddit url support

### ISSUE THREAD

**Issue by openwick** [redlib-org/redlib#202]
## How does this feature work on Reddit?

Users can create  customfeeds and have unique urls and can be viewed and shared with any one like https://www.reddit.com/user/rollidgeli/m/cattos for example. 

## Describe how this could be implemented into Redlib
 Redlib currently doesn't support these urls. Would be great if redlib can scrape these too.

Replace https://www.reddit.com/user/rollidgeli/m/cattos/ with [ Instance url]/user/rollidgeli/m/cattos/ as an example so it can be viewed just like other regular reddit feeds.

## Describe alternatives you've considered
 Currently we can add + to create multireddits but if I want to follow or view  those reddit specific custom feeds we cant.

## Additional context / screenshot

Basically the request summed up in below image. If redlib can support those urls too that would be great.

![image](https://github.com/user-attachments/assets/d3abcee5-0c34-4c0d-8895-8d6422390ee4)

**Comment by gothrek22** [comment]
Kind of a workaround here, and a bit of annoying one, but. If you go to 

https://old.reddit.com/user/rollidgeli/m/cattos,

you can select all of the subreddits that are on the right side. Copy them to a text editor that supports search and replace.

replace all instances of `/r/` with `+`, then just revert the FIRST `+` back to `/r/`, append this new URL to your redlib instance. By the end you should have something like:

```
http://localhost:8080/r/AnimalsBeingDerps+blackcats+cat+catbellies+Catculations+Catloaf+cats+CatsAreAssholes+CatsStandingUp+Catswhoyell+Catswithjobs+CatTaps+jellybeantoes+KneadyCats+OneOrangeBraincell+PeanutWhiskers+PetTheDamnCat+StartledCats+StoppedWorking+SupermodelCats+Thisismylifemeow+toebeans
```

And it you can more or less save that URL as a bookmark, to have a workaround custom feed. Potentially you can also add new subreddits, by appending their name to the url, with a `+` in front.

**Comment by openwick** [comment]
@gothrek22 Thanks but I already knew that. Even mentioned in the description above. 

Just want to have these urls being supported if possible instead of doing all that work :)
